### PR TITLE
Weekly highlights update

### PR DESCRIPTION
**Pull request by taylorbot**
This PR has been created automatically by the 'Create highlight PR' Github workflow
to publish the weekly highlights.

This action is triggered every Thrusday morning to collect all the highlights from the selected apps in giantswarm/github repository. The highlights are collected from the changelog entries from .github repository apps.

Please review the changes, group the entries for each app making sure it is digestible for the customers and highlight the breaking changes in the top.

### CODE REVIEW

**Comment by pipo02mix** [comment]
@Rotfuks can you take a look to the obs apps?

File: src/content/changes/highlights/2024-09-19.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
```
Comment (pipo02mix) [inline src/content/changes/highlights/2024-09-19.md:1]:
```suggestion
```

File: src/content/changes/highlights/2024-09-26.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    - Use `gzip+base64` instead of `base64` for bootstrap data delivery to VM metadata.
+
+- [cluster-vsphere](https://github.com/giantswarm/cluster-vsphere) version [0.64.0](https://github.com/giantswarm/cluster-vsphere/compare/v0.63.0...v0.64.0)
+    - Migrated all worker resources (`KubeadmConfigTemplate`, `MachineDeployment` and `MachineHealthCheck`) to be rendered from the shared `cluster` chart.
+    - Render cleanup hook job using cluster chart.
+
+## Security
+
+- [kyverno-policies-observability](https://github.com/giantswarm/kyverno-policies-observability) version [0.5.0](https://github.com/giantswarm/kyverno-policies-observability/compare/v0.4.0...v0.5.0)
+    - Remove policy for `ServiceMonitor` and `PodMonitor` relabelling schemas as we do not need the enforcement anymore.
```
Comment (stone-z) [inline src/content/changes/highlights/2024-09-26.md:108]: Not sure if this lands here automatically but this is an Atlas topic, not security

File: src/content/changes/highlights/2024-09-26.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+- [kyverno-policies-dx](https://github.com/giantswarm/kyverno-policies-dx) version [0.5.1](https://github.com/giantswarm/kyverno-policies-dx/compare/v0.5.0...v0.5.1)
+    - Use `Enforce` and `Audit` validationFailureAction.
+
+- [kyverno-policies-ux](https://github.com/giantswarm/kyverno-policies-ux) version [0.7.3](https://github.com/giantswarm/kyverno-policies-ux/compare/v0.7.2...v0.7.3)
+    - `cluster-names` now targets Cluster by GVK
+    - Use `Enforce` validationFailureAction.
```
Comment (stone-z) [inline src/content/changes/highlights/2024-09-26.md:115]: Here too, Shield did the work but these are Honeybadger repos

**Review by stone-z — COMMENTED** [review]
This is fine with me, not approving because auto-merge is on and I have comments but if you want to keep the policies all in the same section that's fine with me as well and I'll approve

**Comment by pipo02mix** [comment]
Removed the auto-merge @stone-z

**Review by stone-z — APPROVED** [review]

**Review by pipo02mix — APPROVED** [review]

**Pull request merged**